monitor-cockroachdb-with-prometheus.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Monitor CockroachDB with Prometheus
+summary: 
+toc: false
+---
+
+CockroachDB generates detailed time series metrics for each node in a cluster. This page shows you how to pull these metrics into [Prometheus](https://prometheus.io/), an open source tool for storing, aggregating, and querying time series data. It also shows you how to connect [Grafana](http://grafana.org/) and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to Prometheus for flexible data visualizations and notifications.
+
+<div id="toc"></div>
+
+## Before You Begin
+
+Make sure you have already started a CockroachDB cluster, either [locally](start-a-local-cluster.html) or in a [production environment](cloud-deployment.html). 
+
+## Step 1. Install Prometheus
+
+1. Download the [latest Prometheus tarball](https://prometheus.io/download/) for your OS.
+
+2. Extract the binary and add it to your `PATH`. This makes it easy to start Prometheus from any shell.
+
+3. Make sure Prometheus installed successfully:
+
+   ~~~ shell
+   $ prometheus -version
+   ~~~
+
+   ~~~
+   prometheus, version 1.4.1 (branch: master, revision: 2a89e8733f240d3cd57a6520b52c36ac4744ce12)
+     build user:       root@e685d23d8809
+     build date:       20161128-10:02:41
+     go version:       go1.7.3
+   ~~~
+
+## Step 2. Configure Prometheus
+
+1. Download the starter [Prometheus configuration file](https://github.com/cockroachdb/cockroach/blob/master/monitoring/prometheus.yml) and [aggregation rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/aggregation.rules) for CockroachDB:
+   
+   ~~~ shell
+   # Configuration file:
+   $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/prometheus.yml \
+   -O prometheus.yml
```

Comment:
you don't need `-O prometheus.yml`, wget saves is under the same filename by default.